custom/enikshay/ucr/data_sources/adherence.json

```diff
@@ -0,0 +1,158 @@
+{
+    "domains": [
+        "enikshay-test",
+        "enikshay-test-test",
+        "enikshay",
+        "enikshay-test-2",
+        "enikshay-test-3",
+        "enikshay-nikshay-migration-test",
+        "enikshay-domain-copy-test",
+        "enikshay-aks-audit",
+        "np-migration-3"
+    ],
+    "server_environment": [
+      "softlayer",
```

Comment:
Trailing commas aren't valid in JSON. i'm not sure if python handles this correctly